local-volume/provisioner/pkg/util/volume_util_darwin.go

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
How about calling this _unsupported, and then have a build tag that excludes linux?

Reply:
Done.